spec.bs

```diff
@@ -3716,19 +3727,35 @@ Each {{InterestGroupBiddingScriptRunnerGlobalScope}} has a
 :: A [=boolean=]
 : <dfn>group has ad components</dfn>
 :: A [=boolean=]
+: <dfn>multi-bid limit</dfn>
+:: An {{unsigned short}}.
 
 </dl>
 
+<div algorithm>
+To <dfn>convert one or many GenerateBidOutputs to a list of generated bids</dfn> given a ({{GenerateBidOutput}} or [=sequence=]<{{GenerateBidOutput}}>)
+|oneOrManyBids|, an integer |multiBidLimit|, an [=interest group=] |ig|, a [=currency tag=] |expectedCurrency|, a [=boolean=]
+|isComponentAuction| and a [=boolean=] |groupHasAdComponents|:
+  1. Let |bidSequence| be [=sequence=]<{{GenerateBidOutput}}>.
+  1. If |oneOrManyBids| is a {{GenerateBidOutput}}, then set |bidSequence| to « |oneOrManyBids| ».

```

Comment:
It's a bit more than that, it's basically a union of one GenerateBidOutput or a sequence of GenerateBidOutput, and I want to handle the singleton the same way I would handle a sequence with one entry

Reply:
Sure that makes sense, I'm recommending that you link to specific type instead of say "is a". See https://urlpattern.spec.whatwg.org/#ref-for-dfn-specific-type for example.

Reply:
Oh, didn't realize it was a term (the quotes somehow didn't clue me in). Done, thanks.